config/prod/s3-synapse-sync-bucket-a.yaml

```diff
@@ -26,4 +26,4 @@ parameters:
 # Due to circular dependency enabling bucket notification must be done after bucket creation
```

Comment:
Took me a while to parse this sentence. I think a comma would make it clearer: "Due to circular dependencies, enabling bucket notification must be done after bucket creation"

Reply:
good point @clarisse-lau my enblish is bad.  Awaiting your PR to fix.

Reply:
point made by @cascadianblue , but I agree a comma would make this sentence more clear